components/region_cache_memory_engine/src/background.rs

```diff
@@ -401,11 +442,18 @@ impl Runnable for BackgroundRunner {
                                 }
                             }
                         }
-                        core.on_snapshot_loaded(range).unwrap();
+                        core.on_snapshot_load_finished(range);
                     }
                 };
                 self.range_load_remote.spawn(f);
             }
+            BackgroundTask::MemoryCheck => {
```

Comment:
So the MemoryCheck task is a bit confusing. The real purpose if not checking memory, but in fact to start evict some ranges, right?
Maybe we can rename it as EvictRanges.

Reply:
Maybe we can call CheckAndEvict as we will check the memory.